eva_sub_cli/etc/spreadsheet2json_conf.yaml

```diff
@@ -93,35 +95,35 @@ Sample:
     Taxonomy ID: taxId
     Scientific Name: scientificName
```

Comment:
I know I'm moving the goalpost here, but I'm wondering if we should also align with what we were doing before in [eva-submission](https://github.com/EBIvariation/eva-submission/blob/c39df0abbd3924e857d0490991a86dc77f20d9b3/eva_submission/biosample_submission/biosamples_submitters.py#L409)

The spreadsheet header could either be converted or use has they were.

Reply:
Updated to `scientific name`, and thanks for helping catch `collection date` as well...